packages/genesys-spark-components/src/components/stable/gux-toast/gux-toast.tsx

```diff
@@ -47,6 +47,9 @@ export class GuxToast {
   @State()
   hasSecondaryButton: boolean = false;
 
+  @Prop()
+  hideDismissButton: boolean = false;
+
```

Comment:
What do you think of
```
 @Prop()
  dismissible: boolean = true;
```

Reply:
Yeah I dont mind, I tried to keep it consistent with the `popover` api since its similar. The `popover` has `displayDismissButton`.

Reply:
Ya I just don't like them long names that are referencing elements instead of functionality.
Maybe changing `displayDismissButton` is something we should look int for v5